### PR TITLE
blog: add personal experience report on updating workspaces

### PR DESCRIPTION
**Pull request by skaldarnar**

### CODE REVIEW

File: blog/2023-04-22-waking-up-from-hibernation/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Let me share how I updated my various workspaces when coming back after some time of absence...
+
+## Updating my Workspaces
+
+To start fresh I wanted to reset my workspaces to a clean state - discard any uncommitted changes (at this point I don't have any clue what they might have been about anyways) and pull the latest changes from upstream.
+It's not a hard task and I've done it uncountable times before, but not everybody might be that familiar with the tools and little helpers available to assist with that task.
+
+### Updating my Iota Workspace
+
+To bring my Iota workspace up to speed I'm used the `groovyw' utility tool that is shipped with the engine repository.
```
Comment (jdrueckert) [inline blog/2023-04-22-waking-up-from-hibernation/index.md]:
```suggestion
To bring my Iota workspace up to speed I used the `groovyw` utility tool that is shipped with the engine repository.
```

File: blog/2023-04-22-waking-up-from-hibernation/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+You can run it easily via the provided wrapper scripts, that is `./groovyw` on Linux or `./groovyw.bat` on Windows.
+It comes in handy to initialize a workspace with a specific set of modules, fetch additional modules, or update everything via git.
+Our [Contributor Quick Start Guide](https://github.com/MovingBlocks/Terasology/wiki/Contributor-Quick-Start#set-up-your-terasology-development-workspace) also references it for the initial setup.
+
+But today I'm only interested in updating all my modules in the workspace.
+
+```
+groovyw module update-all
+```
```
Comment (jdrueckert) [inline blog/2023-04-22-waking-up-from-hibernation/index.md:38]: We might want to create an issue to improve formatting here:
image

**Review by jdrueckert — DISMISSED** [review]
Minor correction and if you have time, it would be nice to create an issue for the code block formatting, but I don't see a reason not to merge this after fixing the typos.

File: blog/2023-04-22-waking-up-from-hibernation/index.md (hunk 2 above)
Comment (skaldarnar) [reply blog/2023-04-22-waking-up-from-hibernation/index.md:38]: See #211

**Pull request merged**